integration_tests/testsuite/test_custom.py

```diff
@@ -189,6 +190,12 @@ def _validate(self, context, validation_specification):
                                  'do not match the pattern `{2}`'
                                  .format(value, key, pattern))
 
+        if endpoint:
+            status, _ = test_util.post(url=urlparse.urljoin(self._base_url,
+                                                            endpoint),
+                                       payload=context)
+            self.assertNotEquals('ERROR', status)
```

Comment:
can you make this string a constant in `test_util` and then refer to it here? just in case it ever gets changed

Reply:
It seems that the first parameter of the response is in fact the name of the log to use, so I will switch to the second (response_code).